exercises/concept/annalyns-infiltration/.docs/instructions.md

```diff
@@ -10,7 +10,12 @@ Having found the kidnappers, Annalyn considers which of the following actions sh
 - Stealth attack: if Annalyn reaches the sleeping knight without being detected, she deals 12 damage with a stealth attack. Otherwise, she deals only 7 damage to the knight.
```

Comment:
```suggestion
- _Stealth attack_: if Annalyn reaches the sleeping knight without being detected, she deals 12 damage with a stealth attack. Otherwise, she deals only 7 damage to the knight.
```